packages/taquito-local-forging/src/schema/operation.ts

```diff
@@ -236,7 +236,11 @@ export const schemaEncoder =
         const values = value[key];
 
         if (!Array.isArray(values)) {
-          throw new OperationEncodingError(`Expected value to be Array ${JSON.stringify(values)}`);
+          throw new OperationEncodingError(
+            `Invalid operation value "${JSON.stringify(
```

Comment:
A general concern regarding `JSON.stringify` in errors: If the value cannot be stringified (for instance, if it has circular references), then the error cannot be shown. This makes it very difficult to diagnose the problem.

Maybe we need a `safeStringify` to be used almost everywhere?

https://stackoverflow.com/questions/11616630/how-can-i-print-a-circular-structure-in-a-json-like-format